src/components/AllMovies/AllMovies.js

```diff
@@ -0,0 +1,22 @@
+import './AllMovies.css';
+import Movie from '../Movie/Movie';
+function AllMovies( {movies} ){
+	const moviesCards = movies.map(movie => {
+		return (
+			<Movie 
+				id={movie.id}
+				key={movie.id}
+				posterPath={movie.poster_path}
+				title={movie.title}
+				rating={Math.round(movie.average_rating * 10) / 10}
```

Comment:
Since one of the ratings have large decimal point values, we use Math.round() to make the rating value look nicer.